templates/add-persistence-attribute-SEND_PHONE_MESSAGE/code.js

```diff
@@ -68,3 +62,49 @@ exports.onExecuteSendPhoneMessage = async (event, api) => {
         }
     );
 };
+/**
+ * Get an AccessToken
+ *
+ * @param {CacheAPI} cache
+ * @param {{domain: string, clientId: string, clientSecret: string, audience?: string}} options - AuthenticationClient options to fetch the token
+ * @return {Promise<string>}
+ */
+async function getAccessToken(cache, options) {
+    const key = `access_token_${options.clientId}_${options.audience}`;
```

Comment:
Also, `audience` will always be `undefined`. Maybe we should remove it from the cache-key, OR require it as a secret that's passed in